lib/motion-spec/matcher/raise_error.rb

```diff
@@ -0,0 +1,47 @@
+# -*- encoding : utf-8 -*-
+module MotionSpec
+  module Matcher
+    class RaiseError
+      def initialize(error_class = Exception, message = '')
+        @error_class = error_class.is_a?(Class) ? error_class : Exception
+        @error_message = (error_class.is_a?(String) || error_class.is_a?(Regexp)) ? error_class : message
+      end
+
+      def matches?(value, &block)
+        block.call
+        false
+      rescue Exception => e
```

Comment:
Avoid rescuing the `Exception` class. Perhaps you meant to rescue `StandardError`?